projects/annotation-ui-app/src/app/annotation-webapp/annotation-webapp.component.ts

```diff
@@ -5,6 +5,13 @@ import {Component} from '@angular/core';
     templateUrl: './annotation-webapp.component.html'
 })
 export class AnnotationWebappComponent {
+
+  dmDocumentSelected: boolean = false;
+
     constructor() {
     }
+
+    toggleDocumentSelection() {
+      this.dmDocumentSelected = !this.dmDocumentSelected;
```

Comment:
You should pass the value true or false (depending on which is selected) as when you select DM button twice, it will switch between them.